README.md

```diff
@@ -1 +1,66 @@
-# Module-One-Capstone-Project
\ No newline at end of file
+
+
+# Personal Portfolio project (mobile version)
+
+> This project involves creating a basic mobile version portfolio skeleton which will be used in the development of a complete personal portfolio accessible both through phone and desktop.
+
+## Mobile Version screenshot
+
+![screenshot](./app_screenshot.png)
+
+## Desktop Version screenshot
+
+![screenshot](./app_screenshot2.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML
+- CSS
+- JAVASCRIPT
+
+## Live Demo
+
+[Live Demo Link](https://mosams.github.io/Personal-Portfolio/)
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
- [ ] Make sure to customize this area of your README file as well. If some of the sections don't apply to your project, feel free to remove them.